packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -92,6 +96,26 @@ class CupertinoSwitch extends StatefulWidget {
   /// [CupertinoTheme] in accordance to native iOS behavior.
   final Color activeColor;
 
+  /// {@template flutter.cupertino.switch.dragStartBehavior}
+  /// Determines the way that drag start behavior is handled.
+  ///
+  /// If set to [DragStartBehavior.start], the drag behavior used to move the
+  /// switch from on to off will begin upon the detection of a drag gesture. If
+  /// set to [DragStartBehavior.down] it will begin when a down event is first
+  /// detected.
+  ///
+  /// In general, setting this to [DragStartBehavior.start] will make drag
+  /// animation smoother and setting it to [DragStartBehavior.down] will make
+  /// drag behavior feel slightly more reactive.
+  ///
+  /// By default, the drag start behavior is [DragStartBehavior.start].
+  ///
+  /// See also:
+  ///
+  ///   * {@macro flutter.gestures.monodrag.dragStartExample}
```

Comment:
I don't think dartdoc will render this properly. Instead, just replace this line with:

```
 * [DragGestureRecognizer. dragStartExample], , which gives an example for the different behaviors.
```

Reply:
This also applies to all the other uses below.